src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/MediaBlobContainerTenantEvents.cs

```diff
@@ -10,7 +9,7 @@
 using OrchardCore.Environment.Shell.Removing;
 using OrchardCore.Modules;
 
-namespace OrchardCore.Media.Azure
+namespace OrchardCore.Media.Azure.Services
```

Comment:
file_scope since you already doing modifications here

Reply:
As a reviewer, I find such out-of-scope edits producing whitespace changes for the whole file annoying, but now you're the reviewer and I'm all for file-scoped namespaces :).

Reply:
as a reviewer it's annoying to see the namespace change. But as a submitted, you know you did not make a change, why not.

Reply:
maybe when we have 0 PR, we can use dotnet format to cleanup all the files at once

Reply:
Oh yeah!